doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,218 @@
+            end
+        end
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+We define the boundary conditions similarly to the :doc:`../../incompressible-flow/3d-flow-around-sphere/flow-around-sphere` example although, in our case, we won't need a boundary condition defined on the particle as the sharp immersed boundary method will take care of it.
+
+The domain makes use of an inlet velocity of :math:`1~m/s` on the left, slip boundary conditions parallel to the flow direction, and and an outlet with :math:`p = 0` on the right of the domain. In our case, the outlet boundary condition is the same as the natural boundary condition can therefore be left unchanged as Lethe will apply it by default.
+
+.. code-block:: text
+
+    subsection boundary conditions
+        set number = 5
+        subsection bc 0
+            set id   = 0
+                set type = function
+            subsection u
+                set Function expression = 1
+            end
+            subsection v
+                set Function expression = 0
+            end
+            subsection w
+                set Function expression = 0
+            end
+        end
+        subsection bc 1
+            set id   = 2
+            set type = slip
+        end
+        subsection bc 2
+            set id   = 3
+            set type = slip
+        end
+        subsection bc 3
+            set id   = 4
+            set type = slip
+        end
+        subsection bc 4
+            set id   = 5
+            set type = slip
+        end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+In order to produce a varying Reynolds number, we vary the kinematic viscosity of the fluid. We define the Reynolds number as :math:`Re = \frac{UD}{\nu}` where :math:`U` and :math:`D` are fixed at 1. Therefore, 
+the kinematic viscosity is defined as :math:`\nu = \frac{1}{Re}`. 
+
+Note that this definition of the Reynolds number depends only on the particle's diameter, not its length. We select this parameter because it primarily governs the flow regime around the particle.
+The following block defines the viscosity for a Reynolds number of 1:
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+            set kinematic viscosity = 1
+        end
+    end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~
+
+With the Reynolds number ranging from 0.01 to 5, the simulation can easily be solved by using a steady-state solver. The mesh is sufficiently refined to capture the flow around the particle and therefore doesn't 
+require any mesh adaptation.
+
+.. code-block:: text
+
+    subsection simulation control
+        set method            = steady
+        set output path       = ./particle_drag/
+    end
+
+---------------
+Results
+---------------
+
+We obtain the following solution for the velocity and pressure:
+
+Velocity:
+ 
+.. image:: images/flow-around-particle-sharp-velocity.png
+    :alt: Velocity field around the particle
+    :align: center
```

Comment:
If you've got time to re-do the figures, please use a non-diverging scheme for the colormap. Maybe it could be interesting to visualize the streamlines instead of velocity glyphs.